build/images/Dockerfile.build.windows

```diff
@@ -48,11 +25,11 @@ LABEL description="A Docker image to deploy the Antrea CNI."
 USER ContainerAdministrator
 
 RUN mkdir -Force C:\k\antrea\bin
-COPY --from=cni-binaries-windows  /opt/cni/bin /k/antrea/cni
+COPY --from=antrea-build-windows  /opt/cni/bin /k/antrea/cni
 COPY --from=antrea-build-windows  /antrea/build/images/scripts/Install-WindowsCNI.ps1 /k/antrea/
 COPY --from=antrea-build-windows  /antrea/bin/antrea-agent.exe /k/antrea/bin/
 COPY --from=antrea-build-windows  /antrea/bin/antctl-windows.exe /k/antrea/bin/antctl.exe
 COPY --from=antrea-build-windows  /antrea/bin/antrea-cni.exe /k/antrea/cni/antrea.exe
 
-RUN mkdir C:\k\antrea\utils; \
-    curl.exe -Lo C:/k/antrea/utils/wins.exe https://github.com/rancher/wins/releases/download/v0.0.4/wins.exe
+RUN mkdir C:\k\antrea\utils
+COPY --from=antrea-build-windows /wins/wins.exe /k/antrea/utils/wins/exe
```

Comment:
I think the target file should be `/k/antrea/utils/wins.exe`

Reply:
Updated.